starting-oss-projects.md

```diff
@@ -68,6 +68,8 @@ In addition, the following aspects should be considered in the planning phase:
 
 #### Different Project Levels
 
+TBD: Move out of "project lifecycle" chapter?
```

Comment:
I agree that it feels a bit our of place here. Maybe it would fit into the governance section?

Reply:
Yes, that would fit. I will change it.